app/lib/server/startup/settings.js

```diff
@@ -1303,14 +1303,6 @@ settings.addGroup('Push', function() {
 				_id: 'Push_enable',
 				value: true,
 			},
-			{
```

Comment:
Ok, did it! But with this change I had to modify one validation in `server` > `lib` > `pushConfig.js`, otherwise the custom certificates wouldn't work in unregistered servers.

Reply:
thx @lucassartor I had to change to set an empty `enableQuery` because otherwise the previous value was still set 
on DB: f633ca1aa2f30b88bbe7af83b435025cee240ecc

I also changed a little bit the logic the `configurePush` so it used the same logic in both places 3863792abce8989f3b6c1d498c56d01705d0daa8